bindings/node.js/CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contribution Guidelines
+
+Thanks for your interest in contributing to the Node.js bindings for c-kzg. It's people like you that push the Ethereum ecosystem forward!!
```

Comment:
It's fun, but I think we should remove that second sentence.

```suggestion
Thanks for your interest in contributing to the Node.js bindings for c-kzg-4844.
```

Reply:
Ok.  I committed your suggestion from the PR